quickstarts/microsoft.web/web-app-asp-app-on-asev3-create/modules/privatednszone.bicep

```diff
@@ -5,7 +5,9 @@ param privateDNSZoneName string
 param virtualNetworkId string
 
 @description('Required. ASE network configuration.')
-param aseNetworkConfiguration string
+param aseNetworkConfigurationId string
+
+var aseNetworkConfigurationEndpoint = '${aseNetworkConfigurationId}/configurations/networking'
```

Comment:
use the existing syntax for this, e.g.
```bicep
resource aseConfig 'Microsoft.Web/hostingEnvironments/configurations@2021-01-15' existing = {
name: '${aseName}/networking'
}
```

And then replace the references below with:

```aseConfig.internalInboundIpAddresses[0]```

You'll need to pass in the name of the hosting environment instead of the resourceID

Reply:
tried to use existing syntax for this, however, it seems not able to get internalInboundIpAddresses[0] property. Error as below,

The type "Microsoft.Web/hostingEnvironments/configurations" does not contain property "internalInboundIpAddresses". Available properties include "apiVersion", "eTag", "extendedLocation", "id", "identity", "kind", "location", "managedBy", "managedByExtended", "name", "plan", "properties", "scale", "sku", "tags", "type", "zones".bicep(BCP053)

Reply:
@bmoore-msft, please advise how to make this change? When I change to the existing syntax, `aseConfig.internalInboundIpAddresses[0]` is not available, I can only leave the old syntax on this. Thanks!

Reply:
try:

```aseConfig.properties.internalInboundIpAddresses[0]```

Reply:
It works. Thanks.